services/ymax-planner/src/engine.ts

```diff
@@ -122,6 +122,9 @@ type Powers = {
   cosmosRest: CosmosRestClient;
   signingSmartWalletKit: SigningSmartWalletKit;
   now: typeof Date.now;
+  axelarApiAddress: string;
+  fetch: typeof fetch;
+  log: typeof console.log;
```

Comment:
`startEngine` makes use of both `console.warn` and `console.error`.
```suggestion
  logger: Pick<typeof console, 'warn' | 'error'>;
```